src/flix.js

```diff
@@ -77,66 +77,59 @@ export default function(hljs) {
     };
 
     const KEYWORDS = [
-      "instance",
-      "lawful",
-      "law",
-      "type",
       "alias",
-      "yield",
-      "lazy",
-      "force",
-      "override",
+      "as",
+      "case",
+      "catch",
+      "deafult",
```

Comment:
oops :)

Reply:
(I forget where the default keyword is used).

Reply:
(Its probably fine-- if you decide to do another PR then I would sync with the content of `flix.tmLanguage.json`  from `flix/vscode-flix`.